lessons/pydata/pandas_types/index.ipynb

```diff
@@ -0,0 +1,6941 @@
+{
```

Comment:
<p>Asi by som to trocha preformuloval, aby bolo jasnejší zámer - nie že to nie je praktické, len sa to využíva skôr v špecifických prípadoch (ak to teda zámer bol :-D).</p><p> </p><p>Napáda mi nasledovné:</p><ol><li>Nechať len "Pro hodnoty nového sloupce lze použít i jednu skalární hodnotu:"</li><li>Nechať to, čo v prípade 1, len na koniec pridať "v praxi sa s touto potrebou nestretneme tak často" alebo niečo podobné.</li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PyDataCZ/naucse.python.cz/pull/23/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>